### ISSUE TITLE
Provide a way to change the item's role property in `DropDown.Item`

### ISSUE THREAD

**Issue by matuzalemsteles** [liferay/clay#5004]
In issue #4979, we changed the ownership of `role` to be added in the container and #4998 added the default `role` to the item as `menuitem` to improve the accessibility of DropDown OOTB without the DXP developers having to configure it.

There are some scenarios where the Menu is used as `listbox` and the items should be set to `option`, this conflicts with the item being `menuitem` and the item container being `option`, to improve accessibility here we need leave the container with `role="none"` and the item with `role="option"`.
